src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -33,8 +33,8 @@ public TestService(string serviceName)
 
         public static string GetLogPath(string serviceName)
         {
-            return Assembly.GetEntryAssembly().Location + "." + serviceName + ".log";
-        }        
+            return typeof(TestService).Assembly.Location + "." + serviceName + ".log";
```

Comment:
Did this never work? It was always the other location before.

Reply:
It was previously `typeof(TestService).Assembly.Location` in one side and `Assembly.GetEntryAssembly().Location` in the other. I unified on the wrong one.